tests/unit/sagemaker/workflow/test_functions.py

```diff
@@ -97,3 +99,23 @@ def test_json_get_expressions():
             "Path": "my-json-path",
         },
     }
+
+
+def test_json_get_expressions_without_invalid_step_name():
```

Comment:
nit `test_json_get_expressions_with_invalid_step_name`

Reply:
Good catch! Fixed in new commit.